pulp_ansible/app/models.py

```diff
@@ -65,14 +65,30 @@ class CollectionVersion(Content):
 
     TYPE = "collection_version"
 
+    authors = psql_fields.JSONField(default=list, editable=False)
+    contents = psql_fields.JSONField(default=list, editable=False)
+    description = models.TextField(blank=True, editable=False)
+    documentation = psql_fields.JSONField(default=dict, editable=False)
+    license = psql_fields.JSONField(default=list, editable=False)  # list?
+    name = models.CharField(max_length=32, editable=False)
+    namespace = models.CharField(max_length=32, editable=False)
+    quality_score = models.FloatField(null=True, editable=False)
+    readme = models.TextField(blank=True, editable=False)  # Is this really the filename?
+    tags = psql_fields.JSONField(default=list, editable=False)
```

Comment:
become a separate model. Have it listed in the serializer of CollectionVersion (not it's own viewset and serializer).